pep-0501.txt

```diff
@@ -1,27 +1,45 @@
 PEP: 501
-Title: General purpose string interpolation
-Version: $Revision$
-Last-Modified: $Date$
-Author: Nick Coghlan <ncoghlan@gmail.com>
-Status: Deferred
+Title: General purpose string template literals
```

Comment:
```suggestion
Title: General purpose template literal strings
```

"Template literal strings" is the phrasing used within the PEP body, and seems to make more sense/be easier to read